includes/discovery/fdb-table/jetstream.inc.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * jetstream.inc.php
+ *
+ * Discover FDB data with Q-BRIDGE-MIB for Jetstream switches
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ *
+ * @copyright  2022 Peca Nesovanovic
+ * @author     Peca Nesovanovic <peca.nesovanovic@sattrakt.com>
+ */
+$oids = SnmpQuery::allowUnordered()->hideMib()->walk('Q-BRIDGE-MIB::dot1qTpFdbPort')->table(2);
+if (! empty($oids)) {
+    $insert = [];
+    d_echo('Jetstream: FDB Table');
+    foreach ($oids as $vlan => $oidData) {
+        foreach ($oidData as $mac => $macData) {
+            $port = $macData['dot1qTpFdbPort'];
+            //try both variation with & without space
+            $port_id = find_port_id('gigabitEthernet 1/0/' . $port, 'gigabitEthernet1/0/' . $port, $device['device_id']) ?? 0;
```

Comment:
no chance to have a stack here ? with the "1" hardcoded, won't do good on a stack for sure :)

Reply:
if you look at current situation, this solution is 100% better than current one which is 0% recognized port :(

Reply:
that was an open question :) if you tell me jetstream does not support stacking, then this is good to go. And definitly better than current I can imagine :)

Reply:
https://static.tp-link.com/res/down/doc/T1700G-28TQ(UN)_V1_UG.pdf

Reply:
> https://static.tp-link.com/res/down/doc/T1700G-28TQ(UN)_V1_UG.pdf

Ok mr @murrant 

please look at this:
```

SNMP[.[0;36m'/usr/bin/snmpbulkwalk' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/tplink' '-v2c' '-c' 'COMMUNITY' '-OQXUte' '-Cc' '-Os' 'udp:HOSTNAME:161' 'Q-BRIDGE-MIB::dot1qTpFdbPort'.[0m]

dot1qTpFdbPort[44][74:da:88:4a:ef:ec] = 16
dot1qTpFdbPort[44][a8:5e:45:b5:5f:9c] = 12
dot1qTpFdbPort[100][2:ae:24:96:30:cb] = 24
dot1qTpFdbPort[100][4c:eb:bd:18:c2:4b] = 24
dot1qTpFdbPort[100][9c:7b:ef:5:15:8c] = 24
dot1qTpFdbPort[100][9c:7b:ef:88:f5:10] = 22
dot1qTpFdbPort[200][0:9:0:0:1:3] = 5
dot1qTpFdbPort[200][0:9:0:0:1:5] = 24
```
Ok, Vlan info is in the middle, [xxx], so far so good

but, problem is here:
```
SNMP[.[0;36m'/usr/bin/snmpbulkwalk' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/tplink' '-v2c' '-c' 'COMMUNITY' '-OQXUte' '-Cc' '-Os' 'udp:HOSTNAME:161' 'BRIDGE-MIB::dot1dTpFdbPort'.[0m]
dot1dTpFdbPort[1:0:5f:67:75:df].200 = 49176
dot1dTpFdbPort[1:2:a8:0:5:0].0 = 49176
dot1dTpFdbPort[1:2:a8:0:6:0].1 = 49176
dot1dTpFdbPort[1:2:ae:24:96:30].203 = 49176
dot1dTpFdbPort[1:28:ee:52:f5:92].70 = 49174
dot1dTpFdbPort[1:40:61:86:d:31].0 = 49176
```

this one will give back ifIndex as you see, but only, ONLY for access ports!!! trunk ports are left out

so, could we agreed in next solution ?

1. try to find MAC in bridge-mib (access port)
2. if not, then try to guess (hardcoded 1/0/x)

is this acceptable ?